pkgs/top-level/all-packages.nix

```diff
@@ -15875,7 +15875,7 @@ in
   mariadb = callPackage ../servers/sql/mariadb {
     # As per mariadb's cmake, "static jemalloc_pic.a can only be used up to jemalloc 4".
     # https://jira.mariadb.org/browse/MDEV-15034
-    jemalloc = jemalloc450.override ({ disableInitExecTls = true; });
+    jemalloc = jemalloc450;
```

Comment:
why do you remove the override from here and move it to `pkgs.jemalloc450`? This looks like a no-op to me (for `pkgs.mariadb`), but a breakage for many different packages.

Reply:
jemmalloc450 is not used anywhere.
Use this variant?
```
  mariadb = callPackage ../servers/sql/mariadb {
    # As per mariadb's cmake, "static jemalloc_pic.a can only be used up to jemalloc 4".
    # https://jira.mariadb.org/browse/MDEV-15034
    jemalloc = jemalloc450-mariadb;

...
    jemalloc450 = callPackage ../development/libraries/jemalloc/jemalloc450.nix { };
    jemalloc450-mariadb = callPackage ../development/libraries/jemalloc/jemalloc450.nix { disableInitExecTls = true; };
```

Reply:
If `jemalloc450` isn't used anywhere, I'd propose to do the `callPackage` to `jemalloc450.nix` entirely inside the `mysql` derivation.

Reply:
Still, I don't see how your PR changes anything. You just move the override from one process to another - the same derivation is passed as `jemalloc` to the `mysql` derivation.

Reply:
Whereas it is correct to load the library with `disableInitExecTls = true;` ?
```
malloc-lib = ${pkgs.jemalloc450}/lib/libjemalloc.so
```

Reply:
If you only `callPackage` of `jemalloc450.nix` inside the `mariadb`, you can call it with `disableInitExecTls = true;`, yes.